src/javascript/app/pages/user/new_account/virtual_acc_opening.js

```diff
@@ -15,6 +15,7 @@ const urlFor                   = require('../../../../_common/url').urlFor;
 const Utility                  = require('../../../../_common/utility');
 const isEuCountrySelected      = require('../../../../_common/utility').isEuCountrySelected;
 const isBinaryApp              = require('../../../../config').isBinaryApp;
+const ClientBase               = require('../../../../_common/base/client_base');
```

Comment:
you can put this at line 17 along with other common utility

Reply:
done